printer_mods/richardjm/picam-corner/README.md

```diff
@@ -0,0 +1,13 @@
+# Picam corner mount
+
+About the only place I found to mount the picam with a view low down
+of the bed. This was inspired by the [v0 design by xbst](https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/xbst_/V0_Picam)

```

Comment:
Can you make this a relative link?

    [v0 design by xbst](../../xbst_/V0_Picam)

Reply:
No problems, done.